CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+## First steps
+Thank you for getting involved! Before you start, please make sure to familiarize yourself with our README.md
+
+Please take note of our Code of Conduct https://github.com/edgelesssys/edgelessrt/blob/master/CODE_OF_CONDUCT.md and follow it when interacting with our project.
+
+If you want to support our development:
+*	Add a GitHub Star to the project
+*	Share our projects on Social Media
+*	Join out Discord at https://discord.com/invite/rH8QTH56JN
+
+Edgeless RT is licensed under the [MIT License](https://github.com/edgelesssys/edgelessrt/blob/master/LICENSE). When contributing, you also need to agree to our [Contributor License Agreement](https://cla-assistant.io/edgelesssys/edgelessrt).
+
+## Development philosophy
+
+As coding guidelines, we use openenclave's [Development Guide](https://github.com/openenclave/openenclave/blob/master/docs/Community/DevelopmentGuide.md).
```

Comment:
Those seem very comprehensive. There are also more DOs and DON'Ts in their Contributing.md. Should we reference those too, or still maintain our own list? I personally don't feel like that general coding guidelines such as "make your code small/readable/secure" make sense. Obviously, everything depends on context and some general code quality can be expected.

The only thing that maybe needs to be mentioned, is the need for unit tests.

Reply:
For ERT, I agree to limit this to their guide. We can omit the unit test requirement until we have contributions.